templates/memcached-Debian.conf.j2

```diff
@@ -12,17 +12,26 @@ logfile {{ memcached_log_file }}
 # memory
 -m {{ memcached_memory_limit }}
 
-# Default connection port is 11211
--p {{ memcached_port }}
-
 # Run the daemon as root. The start-memcached will default to running as root if no
 # -u command is present in this config file
 -u {{ memcached_user }}
 
+{% if memcached_use_socket %}
+
+-s {{ memcached_socket_path }}
```

Comment:
It looks like there's no default added for `memcached_socket_path`, which would cause a failure if `memcached_use_socket` is true.